killrvideo/__init__.py

```diff
@@ -49,8 +53,10 @@ def serve():
             time.sleep(10)
 
     # Initialize Cassandra Driver and Mapper
-    profile = ExecutionProfile(consistency_level = ConsistencyLevel.LOCAL_QUORUM)
-    cluster = Cluster(contact_points=['10.0.75.1'], execution_profiles={EXEC_PROFILE_DEFAULT: profile})
+    profile = ExecutionProfile(consistency_level =
+                               ConsistencyLevel.name_to_value[default_consistency_level])
```

Comment:
Just to make sure I understand. In the config file you would enter in a string value for the consistency level?

Reply:
yes, text value